src/components/Movies.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import "../App.css";
+
+
+class Movies extends React.Component {
+    constructor() {
+        super()
+
+        this.state = {
+            movie: [],
+            movieTitle: null,
+            movieDate: null,
+            movieDescrip: null
+        }
+    }
+
+    fetchPeople = () => {
+        fetch('https://ghibliapi.herokuapp.com/films')
+        .then((res) => res.json())
+        .then((data) => {
+            this.setState({
+                movie: data,
+            })
+        })
+    }
+
+    componentDidMount = () => {
+        this.fetchMovie()
+    }
+
+    handledropDownChange = (event) =>{
+        this.setState({
+            movieTitle: event.target.value,
+        })
+    }
+
+  render (){
+     
+      let dropDownOption =this.state.movie.map((movie) => {
+    return <option value='movie.title'>{movie.title}</option>;
+})
+
+    let currentMovieObject = this.state.movie.find((movie) => {
+        return movie.title2 === this.state.currentMovie
+    })
```

Comment:
```suggestion
    let currentMovieObject = this.state.movie.find((movie) => {
        return movie.title === this.state.movieTitle
    })
```
You were so close to getting this to work! But there are two more typos here that are keeping your code from working properly. Each `movie` object has a `title` key, not a `title2` key... similarly, `currentMovie` doesn't exist. You meant to put `this.state.movieTitle`.